test/studies/ml/performance/mnist_cnn_batch_verbose.chpl

```diff
@@ -0,0 +1,124 @@
+import Chai as chai;
+import Tensor as tn;
+use Tensor;
+import Math;
+import MNIST;
+import Random;
+import IO;
+import Time;
+
+config param perfTest = false;
+
+tn.seedRandom(0);
+
+config const dataPath = "./data";
+
+var net = new chai.Network(
+    (
+        new chai.Conv(1,8,4,stride=2),
+        new chai.Conv(8,12,5),
+        new chai.MaxPool(),
+        new chai.SoftMax(10)
+    )
+);
+
+
+proc forward(x: Tensor(?), lb: int) {
+    const output = net.forwardProp(x);
+    const loss = -Math.log(output[lb]);
+    const acc = tn.argmax(output.data) == lb;
+    return (output,loss,acc);
+}
+
+proc train(data: [] (Tensor(3),int), lr: real = 0.005) {
```

Comment:
Why doesn't this test use the `train` proc in MNISTTester.chpl?

Reply:
This file is just to show how the network would be trained without using those helper functions that are specific to these MNIST tests. Should I get rid of it and consolidate?

Reply:
Yeah, I'd use the same training procedure in all the tests if possible.